.github/workflows/auto-pr.yml

```
@@ -46,6 +46,14 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
+      - name: Fetch Latest Commit Message from uat
+        env:
+          GITHUB_TOKEN: ${{ secrets.SECRET_GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:48: Double quote to prevent globbing and word splitting

(shellcheck)